### PR TITLE
Lite Magnet Rebalancing

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## About the PR

- Magnet pulls come in a lot closer
- Changed weights for pull types to make wrecks and asteroids more common
- Make equipment spawners give better loot on average
- Make carp spawners weighted more towards single carp than groups or sharks
- carp and sharks are a bit slower
- carps and sharks have slightly less health and do a bit less damage
- plasma and uranium asteroids are more common
- rarer ore groups on asteroid pulls are a bit larger per vein.
- scrap is more common but weighted less towards big scrap

## Why / Balance

It was a bit too punishing overall and not really rewarding enough.

## Requirements

- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Space Carp and Sharkminnows are a bit weaker overall.
- tweak: The magnet now pulls in debris a bit closer.
- tweak: Space debris now has slightly better loot.

### CODE REVIEW

**Comment by Awlod** [comment]
😺👍🎇🎇🎇🎇

**Comment by Sarahon** [comment]
Thank god.

**Comment by Anzarot121** [comment]
Dragon nerf real

File: Resources/Prototypes/Entities/Mobs/NPCs/carp.yml [hunk 1]
```diff
@@ -39,7 +39,10 @@
     - type: MobThresholds
       thresholds:
         0: Alive
-        50: Dead
+        40: Dead
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Mobs/NPCs/carp.yml:42]: carp changes should be to the salvage one, this affects dragon traitors etc

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Pull request merged**